autospec/specfiles.py

```diff
@@ -730,6 +730,46 @@ def write_maven_install(self):
         # All done
         self._write_strip("done")
 
+    def write_gradle_install(self):
+        """Write installation steps to specfile for maven source packages."""
+        self.write_license_files()
+
+        if self.subdir:
+            self._write_strip("pushd " + self.subdir)
+
+        self._write_strip("gradle --offline install -d")
+
+        # Enter the installation subdir
+        self._write_strip("pushd build/install")
+
+        # Handle jarfiles in lib dir
+        self._write_strip("shopt -s nullglob")
+        self._write_strip("for lib_dir in */lib; do")
+        self._write_strip("echo BTW OK 2")
+        self._write_strip("pushd ${lib_dir}")
+        self._write_strip("for jarfile in *.jar; do")
+        self._write_strip("JARNAME=$(basename ${jarfile})")
+        self._write_strip('REALJAR=$(find /usr/share/java/.m2/repository -type f -name "${JARNAME}" | head -1)')
```

Comment:
I'm trying to understand the ` | head -1` bit. If there is more than jar file with that name in the repository, how do we know we have chosen the correct one?

On the other hand, if there is only one possible match for the jar file name in the repository, then why is ` | head -1` needed?

Reply:
This was a leftover from an attempt where I searched multiple directories and wanted to prioritize the first result. But yeah, it would be better to fail here if we get multiples.